packages/design-tokens-schema/src/base-token.ts

```diff
@@ -39,6 +39,10 @@ export const BaseDesignTokenValueSchema = z.strictObject({
 });
 export type BaseDesignTokenValue = z.infer<typeof BaseDesignTokenValueSchema>;
 
+const BaseDesignTokenPropertiesEnum = BaseDesignTokenValueSchema.keyof();
+export type BaseDesignTokenProperty = z.infer<typeof BaseDesignTokenPropertiesEnum>;
+export const BaseDesignTokenProperties = BaseDesignTokenPropertiesEnum.options;
```

Comment:
```suggestion
export const baseDesignTokenProperties = BaseDesignTokenPropertiesEnum.options;
```
or 
```suggestion
export const BASE_DESIGN_TOKEN_PROPERTIES = BaseDesignTokenPropertiesEnum.options;
```

The uppercase B confused me a bit down the line